tests/xfstests/enable_kdump.pm

```diff
@@ -34,7 +34,12 @@ sub run {
 
     # Activate kdump
     prepare_for_kdump;
-    activate_kdump;
+    if (check_var('ARCH', 'x86_64')) {
```

Comment:
In this way the code does make it totally non-obvious why you have two different branches. Why should we differ by architecture?

Reply:
Because ppc64le and s390x are double page size than x86_64. Different branches has different suitable "remain memory", to set them all to 72,174 is not a good choice. From following snapshot(not focus area) you can see, yast choose different RAM size for them. https://openqa.suse.de/tests/2303087#step/enable_kdump/30  https://openqa.suse.de/tests/2303416#step/enable_kdump/30

Reply:
As aarch64 ppc64le are working well with the kdump enable process, and different workers has tiny different RAM size, so I prefer to keep use yast to set those RAM size for them.

Reply:
can you try to summarize this two messages in a comment in file which would explain same for others who don't have chance to read this PR ?

Reply:
@richiejp consider the links I give, different machine has different alloc_mem set by yast. It looks like a magic number for me. It may also infect by something other than pagesize, then I leave aarch64 and ppc64le for yast to set. :(